modules/ossm-security-cert-manage-1x.adoc

```diff
@@ -0,0 +1,169 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v1x/ossm-security.adoc
+
+
+[id="ossm-cert-manage_{context}"]
+= Adding an external certificate authority key and certificate
+
+By default, {ProductName} generates self-signed root certificate and key, and uses them to sign the workload certificates. You can also use the user-defined certificate and key to sign workload certificates, with user-defined root certificate. This task demonstrates an example to plug certificates and key into {ProductShortName}.
+
+.Prerequisites
+
+* You must have installed {ProductName} with mutual TLS enabled to configure certificates.
+* This example uses the certificates from the link:https://github.com/maistra/istio/tree/maistra-2.0/samples/certs[Maistra repository]. For production, use your own certificates from your certificate authority.
+* You must deploy the Bookinfo sample application to verify the results with these instructions.
+
+[id="ossm-cert-manage-add-cert-key_{context}"]
+== Adding an existing certificate and key
+
+To use an existing signing (CA) certificate and key, you must create a chain of trust file that includes the CA certificate, key, and root certificate. You must use the following exact file names for each of the corresponding certificates. The CA certificate is called `ca-cert.pem`, the key is `ca-key.pem`, and the root certificate, which signs `ca-cert.pem`, is called `root-cert.pem`. If your workload uses intermediate certificates, you must specify them in a `cert-chain.pem` file.
+
+Add the certificates to {ProductShortName} by following these steps. Save the certificates from the link:https://github.com/maistra/istio/tree/maistra-2.0/samples/certs[Maistra repo] locally and replace `<path>` with the path to your certificates.
```

Comment:
Maybe we should make it clear that these files are strictly examples? It sounds a bit like using these was required